### PR TITLE
 ✨  update the layout and look and feel of the ios example app to use

### PR DESCRIPTION
**Pull request by wei-lee**
the drawer menu pattern. Add the initial auth view

### Motivation

* We want the ios example app to be like the android example app
* We need some view to test the iOS IDM SDK

### Description

* Update the layout of the app to use the drawer menu pattern
* Reorganise the structure of the app so that each feature will have its own folder to group all the code together, include UI, storyboard etc 

### Progress

- [x] Update the layout 
- [x] Add auth view

### Additional Notes

To test, build the app and run it emulator. Then try the authentication view.

### CODE REVIEW

**Comment by wei-lee** [comment]
ping @aidenkeating @rachael-oregan @TommyJ1994 @ziccardi

**Comment by wtrocki** [comment]
We went here the officially documented apple way for UI using single storyboard, which is fine, but simpler aproach for other developers (any more popular in IOS dev now) will be to use separate xib files for each view.

**Comment by wtrocki** [comment]
@wei-lee - Changes look ok. I'm assuming that this was generated from XCode template and extented later. I have no means to verify that now so leaving that to other members.

**Comment by wei-lee** [comment]
@wtrocki I did add you as a reviewer but then click on it and didn't realise it will remove you from the reviewer list. That is just a mistake. Sorry again.

> I'm assuming that this was generated from XCode template and extented later.

Not really generated by xcode template. But does it matter?

File: example/AeroGearSdkExample/RootViewController.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
+//
+//  RootViewController.swift
+//  AeroGearSdkExample
+//
+//  Created by Wei Li on 26/02/2018.
```
[the remark this answers is not in the document]
Comment (wei-lee) [reply example/AeroGearSdkExample/RootViewController.swift]: yeah, will do

**Comment by wei-lee** [comment]
ping @danielpassos do you mind taking a look and approve the PR if that is ok? @wtrocki is on PTO.

**Comment by danielpassos** [comment]
@wei-lee In my queue

**Comment by danielpassos** [comment]
@wei-lee Can you fix the build before merge it?

**Comment by wtrocki** [comment]
Reviewed. Fixed build. Awesome job!

> Not really generated by xcode template. But does it matter?

I got impression that code was generated and do not need line by line review, but after checking details I see it's not. It's really really good job!

**Pull request merged**